specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2025-03-01-preview/Issues.json

```diff
@@ -0,0 +1,1248 @@
+        }
+      ]
+    },
+    "ListParameter": {
+      "type": "object",
+      "description": "Parameters for listing related entities",
+      "properties": {
+        "filter": {
+          "type": "string",
+          "description": "The filter to apply on the operation. For example, to filter by relevance, use \"$filter=relevance eq 'Relevant'\""
+        }
+      }
+    },
+    "Origin": {
+      "type": "object",
+      "description": "Details about the origin of the entity - the source that added it to the issue",
+      "properties": {
+        "addedBy": {
+          "type": "string",
+          "description": "The ID of the origin - for example, in case of 'Manual', the object ID of the identity, and in case of 'Automatic', the name of the automatic system"
+        },
+        "addedByType": {
+          "$ref": "#/definitions/AddedByType",
+          "description": "The source of the origin - Manual or Automatic"
+        }
+      },
+      "required": [
+        "addedBy",
+        "addedByType"
+      ]
+    },
+    "PagedRelatedAlert": {
+      "type": "object",
+      "description": "Paged collection of RelatedAlert items",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The RelatedAlert items on this page",
+          "items": {
+            "$ref": "#/definitions/RelatedAlert"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "PagedRelatedResource": {
+      "type": "object",
+      "description": "Paged collection of RelatedResource items",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The RelatedResource items on this page",
+          "items": {
+            "$ref": "#/definitions/RelatedResource"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "RelatedAlert": {
+      "type": "object",
+      "description": "Properties of an alert which is related to the issue",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "arm-id",
+          "description": "The alert ID"
+        },
+        "relevance": {
+          "$ref": "#/definitions/Relevance",
+          "description": "The alerts's relevance status"
+        },
+        "origin": {
+          "$ref": "#/definitions/Origin",
+          "description": "The source that related the alert to the issue",
+          "readOnly": true
+        },
+        "addedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time this relation was added to the issue",
```

Comment:
can you update the description to mention the timezone ? here and other places as needed.

Reply:
Done